gcp_variant_transforms/transforms/merge_headers_test.py

```diff
@@ -305,8 +306,9 @@ def test_combine_pipeline(self):
     headers_1 = self._get_header_from_reader(vcf_reader_1)
     headers_2 = self._get_header_from_reader(vcf_reader_2)
 
-    header_merger = HeaderMerger(HeaderConflictResolver(
-        split_alternate_allele_info_fields=True))
```

Comment:
Out of curiosity, do you know why all the tests in this file except this one, directly use the the methods of the `_MergeHeadersFn` instead of using TestPipeline? It seems to me that tests for a PTransform should preferably test its public interface not how it is implemented. I have not checked the details and I think you are not the original author of this but I thought maybe you know.

Reply:
I don't know :-).

Reply:
Added a TODO to unify tests. 

@arostamianfar : let us know if there is any reason, they should stay as they are.

Reply:
I agree with Bashir. Thanks for adding the TODO!